README.rst

```diff
@@ -12,8 +12,8 @@ The Docker image expects the following environment variables to be set:
 
 ``ALLOWED_REMOTE_NETWORKS``
     List of IP networks (CIDR) the bastion host is allowed to access.
-``GRANTING_SERVICE_SSH_KEY``
-    The public SSH key for the "granting-service" user.
+``GRANTING_SERVICE_SSH_KEYS``
```

Comment:
You changed the Variable only in the readme

Reply:
Oops, fixed.